packages/providers/src/fetch_json.ts

```diff
@@ -60,10 +60,12 @@ export async function fetchJsonRpc(url: string, json: JsonRpcRequest, headers: o
             throw new ProviderError(await res.text(), { cause: status });
         }
 
-        if (status === 503) {
```

Comment:
What is the reason for removing `503` error? It is still the case as I see in the `nearcore` code. It should be retried as well